hack/logcheck.conf

```diff
@@ -0,0 +1,22 @@
+# This file contains regular expressions that are matched against <pkg>/<file>,
+# for example k8s.io/cmd/kube-scheduler/app/config/config.go.
+#
+# By default, structured logging call parameters are checked, but usage of
+# those calls is not required. That is changed on a per-file basis.
+#
+# Remember to clean the golangci-lint cache when changing the configuration and
+# running the verify-golangci-lint.sh script multiple times, otherwise
+# golangci-lint will report stale results:
+#    _output/local/bin/golangci-lint cache clean
+
+# At this point we don't enforce the usage structured logging calls except in
+# those packages that were migrated. This disables the check for other files.
+-structured *
```

Comment:
Is just a `*` correct regexp to match any? Should it not be `.*`?

Reply:
True.

Compiling that regexp should have failed. Let me check that.

/hold

Reply:
`*` fails to compile, but logcheck surrounds and `^*$` compiles. `^(*)$` then fails again. The latter is probably what I'll use.

Reply:
@serathius : can you merge https://github.com/kubernetes/klog/pull/315?

Reply:
d

Reply:
Fixed and PR updated.

Reply:
/hold cancel